navigation/MainTabNavigator.js

```diff
@@ -106,392 +110,373 @@ import ReservationFormScreen from '../screens/ReservationFormScreen'
+      <Stack.Screen name="EinvoiceStatusScreen" component={EinvoiceStatusScreen} />
+      <Stack.Screen name="EinvoiceEditScreen" component={EinvoiceEditScreen} />
+      <Stack.Screen name="NewOffer" component={NewOffer} />
+      <Stack.Screen name="EditOffer" component={EditOffer} />
+      <Stack.Screen name="ProductsOverviewforOffer" component={ProductsOverviewforOffer} />
+      <Stack.Screen name="SubscriptionScreen" component={SubscriptionScreen} />
+      <Stack.Screen name="MemberScreen" component={MemberScreen} />
+      <Stack.Screen name="MemberFormScreen" component={MemberFormScreen} />
+    </Stack.Navigator>
+  );
+}
 
-Reservations.navigationOptions = ({screenProps: {t}}) => ({
-  title: t('menu.Reservation'),
-  tabBarButtonComponent: (props) => (
-    testVisible ?
-      <TabBarIcon focused={props?.focused} name="calendar-check" iconLib="MaterialCommunityIcons" onPress={() => NavigationService?.navigateToRoute('ReservationScreen', null, props?.onPress)} />
-      : null
-  ),
-})
+const TablesStack = () => {
+
+  return (
+    <Stack.Navigator
+      initialRouteName="TablesScr"
+      headerMode="screen"
+      screenOptions={{headerShown: false}}
+    >
+      <Stack.Screen name="TablesScr" component={TablesScreen} options={({route: {params}}) => ({
+        animationEnabled: params?.withAnimation ? false : true
+      })} />
+      <Stack.Screen name="OrderDisplayScreen" component={OrderDisplayScreen} />
+      <Stack.Screen name="OrderStart" component={OrderStart} />
+      <Stack.Screen name="NewOrderForm" component={OrderForm} />
+      <Stack.Screen name="OrderFormII" component={OrderFormII} />
+      <Stack.Screen name="OrderFormIII" component={OrderFormIII} />
+      <Stack.Screen name="OrderFormIV" component={OrderFormIV} />
+      <Stack.Screen name="OrdersSummary" component={OrdersSummary} />
+      <Stack.Screen name="UpdateOrder" component={UpdateOrder} />
+      <Stack.Screen name="Payment" component={Payment} />
+      <Stack.Screen name="PaymentOrder" component={PaymentOrder} />
+      <Stack.Screen name="CheckoutComplete" component={CheckoutComplete} />
+      <Stack.Screen name="SpiltBillScreen" component={SpiltBillScreen} />
+      <Stack.Screen name="SplitBillByHeadScreen" component={SplitBillByHeadScreen} />
+    </Stack.Navigator>
+  );
+}
 
-class Hidden extends React.Component {
-  render() {
-    return null;
-  }
+const OrdersStack = () => {
+  return (
+    <Stack.Navigator
+      initialRouteName="OrdersScr"
+      headerMode="screen"
+      screenOptions={{headerShown: false}}
+    >
+      <Stack.Screen name="OrdersScr" component={OrdersScreen} options={({route: {params}}) => ({
+        animationEnabled: params?.withAnimation ? false : true
+      })} />
+      <Stack.Screen name="OrderDetail" component={OrderDetail} />
+      <Stack.Screen name="UpdateOrderFromOrderDetail" component={UpdateOrder} />
+      <Stack.Screen name="NewOrderForm" component={OrderForm} />
```

Comment:
Does this belong to this stack?

Reply:
The old list is:
OrdersScr: OrdersScreen,
OrderDetail: OrderDetail,
UpdateOrderFromOrderDetail: UpdateOrder,
NewOrderForm: OrderForm

so copy for new container